vhostuser/vhostuser.go

```diff
@@ -74,10 +73,10 @@ func loadConf(bytes []byte) (*NetConf, error) {
 }
 
 // saveVhostConf Save the rendered netconf for cmdDel
-func saveVhostConf(conf *NetConf, containerID string) error {
-	fileName := fmt.Sprintf("%s-%s.json", containerID[:12], conf.If0name)
+func saveVhostConf(conf *NetConf, args *skel.CmdArgs) error {
```

Comment:
I'm not sure that we should be passing all of the command line arguments here. It's a bit cleaner to add ifname as another argument to make the function definition more precise.